packages/docs/content/guidelines/exemptions.mdx

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
The exemptions pages are now living on Confluence and can be deleted from here! :partying\_face: